### PR TITLE
jekyll site service times wire-up

### PR DESCRIPTION
**Pull request by jacobtellep**
## Problem
We need a centralized place for service times throughout crds sites.

## Solution
Wire up service times from contentful for the locations pages

### Corresponding Branch
*Add link to crdschurch/repo-name-here#issue-number (if applicable). This helps the code reviewer know that corresponding work exists and where to find it.*

## Testing
*Include information on how to test your work here (if applicable).*

## Associated PRs

- https://github.com/crdschurch/crds-unified/pull/1296
- https://github.com/crdschurch/crds-components/pull/2071

### CODE REVIEW

File: _includes/_location-info.html [hunk 1]
```diff
@@ -2,24 +2,52 @@
...
+        {% endif %}
+    
+        {% if special_service_time_label %}
+          <span class="font-family-base-bold">{{ special_service_time_label }}</span><br>
+        {% endif %}
+        {% for service_time in special_service_times %}
+          {{ service_time }}
+          <!-- don't render the <br> if it's the last line rendered -->
+          {% if forloop.index != special_service_times.size %}<br>{% endif %}
         {% endfor %}
```
Comment (jodylane) [inline _includes/_location-info.html:48]: one issue I see here with this solution is that there was a desire to have the order of the service times display change based on the order service times were added on the location.

**Review by jodylane — CHANGES_REQUESTED** [review]

**Comment by sscarberry** [comment]
Got a little boo boo here on the /locations pages:
image
image

Also we missed the service times block on the bespoke locations template i.e. /oakley
image

**Comment by sscarberry** [comment]
Additional to my comment - it looked like the changes are made, but not working correctly.

File: _includes/_location-info.html [hunk 2]
```diff
@@ -2,24 +2,38 @@
...
-            <div>{{ parts[0] }} {{ parts[1] | replace: '%2C', ',' }}{% unless forloop.last %}<br>{% endunless %}</div>
-          {% else %}
-            <div>{{ parts[0] }}: {{ parts[1] | replace: '%2C', ',' }}{% unless forloop.last %}<br>{% endunless %}</div>
-          {% endif %}
-        {% endfor %}
-      {% endif %}
-      {% if page.service_times %}
-        {{ page.service_times | markdownify | remove: '<p>' | remove: '</p>' }}
-      {% endif %}
+        {% assign current_date = "now" | date: "%s" %}
```
Comment (sscarberry) [inline _includes/_location-info.html:6]: These dates don't appear to be rendering correctly based on contentful config

**Review by sscarberry — CHANGES_REQUESTED** [review]

**Review by jodylane — APPROVED** [review]
this appears to be functioning

**Review by sscarberry — APPROVED** [review]

**Pull request merged**